src/App.tsx

```diff
@@ -1,61 +1,127 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { ChangeEvent, FormEvent, useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+export const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
 
 export const App = () => {
+  const [selectUserId, setselectUserId] = useState(0);
+  const [title, setTitle] = useState('');
+  const [listTodos, setListTodos] = useState<Todo[]>(todos);
+
+  const [hasNameError, setHasNameError] = useState(false);
+  const [hasTitleError, setHasTitleError] = useState(false);
+
+  const valideData = `zxcvbnmasdfghjklqwertyuiopZXCVBNMASDFGHJKLQWERTY
+  UIOPячсмитьбюЯЧСМИТЬБЮфывапролджэФЫВАПРОЛДЖЭйцукенгшщзхъЙЦУКЕНГШЩЗХёЁ123456789`;
+
+  const resetForm = () => {
+    setTitle('');
+    setselectUserId(0);
+  };
+
+  const selectInput = (event: ChangeEvent<HTMLSelectElement>) => {
+    setselectUserId(+event.target.value);
+    setHasNameError(false);
+  };
+
+  const handleInput = (event: ChangeEvent<HTMLInputElement>) => {
+    if (event.target.value
+      .split('')
+      .every(letter => valideData.includes(letter))) {
+      setTitle(event.target.value);
+      setHasTitleError(false);
+    }
+  };
+
+  const sendForm = (event: FormEvent<HTMLFormElement>) => {
```

Comment:
this function doesn't send any form
it's just invoked on form submit, so better to name it as some handler: `handleFormSubmit`, for example